deps/build.jl

```diff
@@ -159,22 +159,30 @@ if !libmxnet_detected
           `git submodule update --init --recursive`
           `git -C mshadow checkout -- make/mshadow.mk`
           `make clean`
+          `cp ../../cblas.h include/cblas.h`
+
           `sed -i -s "s/MSHADOW_CFLAGS = \(.*\)/MSHADOW_CFLAGS = \1 $ilp64/" mshadow/make/mshadow.mk`
-        end
-        FileRule(joinpath(_mxdir, "config.mk"), @build_steps begin
-          ChangeDirectory(_mxdir)
+
+          # Copy config.mk, always override the file
           if is_apple()
             `cp make/osx.mk config.mk`
           else
             `cp make/config.mk config.mk`
           end
+
+          # Configure OpenCV
           `sed -i -s 's/USE_OPENCV = 1/USE_OPENCV = 0/' config.mk`
+
+          # Configure CUDA
           if HAS_CUDA
-            `sed -i -s 's/USE_CUDA = 0/USE_CUDA = 1/' config.mk`
-            if haskey(ENV, "CUDA_HOME")
-              `sed -i -s 's/USE_CUDA_PATH = NULL/USE_CUDA_PATH = $(ENV["CUDA_HOME"])/' config.mk`
+            @build_steps begin
+              `sed -i -s 's/USE_CUDA = 0/USE_CUDA = 1/' config.mk`
+              if haskey(ENV, "CUDA_HOME")
+                `sed -i -s "s@USE_CUDA_PATH = NONE@USE_CUDA_PATH = $(ENV["CUDA_HOME"])@" config.mk`
```

Comment:
I have to admit I don't know `sed` well enough. Have you tested this locally?

Reply:
yes, I tested it.

Let me show you some `sed` usage: we can use `@` as delimiter to deal with something like path.
```sh
└─[iblis@kaladbolg]% echo 'hello sed' | sed -e "s@hello@/path/contains/slash@"                      
/path/contains/slash sed
```

Reply:
Okay cool. LGTM